### DISCUSSION TITLE
Consuming Multicast - Unsupported Transport

### DISCUSSION THREAD

**Discussion by domborovicova** [blakeblackshear/frigate#1167] [General]
Hello,
I tried to consume multicast feed from my synology surveillance station however frigate nvr cannot setup camera for this method.

`ERROR   : rtsp://syno:<hash>@<ip>:554/Sms=4.multicast: Protocol not supported`
`ERROR   : [rtsp @ 0x5595eb0e5900] method SETUP failed: 461 Unsupported Transport`

Can Frigate actually consume multicast streams ? 

Thank you

**Comment by blakeblackshear** [comment]
If ffmpeg can consume multicast streams, then so can frigate. It's just a matter of adjusting the input args which assume a TCP connection.

**Reply by domborovicova** [reply]
Thank you for clarification :) I will need to find proper input arguments to make it work.
What confuses me is that in VLC everything works on first try although loading is a bit long.
